JPS_Ontology/KBTemplates/TBox/OntoSewage.csv

```diff
@@ -0,0 +1,169 @@
+PropertyConnection,Class,EndpointType,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Property connection
+StreetInlet,Class,EndpointType,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Street inlet
+ManholeEndPoint,Class,EndpointType,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Manhole end point
+BranchConnection,Class,SewerageComponent,IS-A,,,,"means taking a tapping connection from a run pipe directly by making a hole in the run pipe or using an outlet fitting, which provides a connection of pipe for fluid to flow from a higher diameter to a lower diameter of the pipe.",https://www.theworldavatar.com/kg/ontosewage/,Branch connection
+http://www.ontology-of-units-of-measure.org/resource/om-2/Distance,Class,,,,,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,
+https://saref.etsi.org/saref4watr/Manhole,Class,SewerageComponent,IS-A,,,,,https://saref.etsi.org/saref4watr/,
+NodeType,Class,,,,,,is the type of a manhole.,https://www.theworldavatar.com/kg/ontosewage/,Node type
+OutletNodeType,Class,NodeType,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Outlet node type
+StructureNodeType,Class,NodeType,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Structure node type
+StreetInletNodeType,Class,NodeType,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Street inlet node type
+FictitiousManholeNodeType,Class,NodeType,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Fictitious manhole node type
+BuildingConnection,Class,NodeType,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Building connection
+InspectionOpening,Class,NodeType,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Inspection opening
+ManholeNodeType,Class,NodeType,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Manhole node type
+ConnectionType,Class,,,,,,is the type of the node connected to a manhole.,https://www.theworldavatar.com/kg/ontosewage/,Connection type
+StreetInletConnectionType,Class,ConnectionType,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Street inlet connection type
+FictitiousManholeConnectionType,Class,ConnectionType,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Fictitious manhole connection type
+StandardShaftCircularProfile,Class,ConnectionType,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,standard shaft circular profile
+RectangularShaft,Class,ConnectionType,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Rectangular shaft
+Cover,Class,,,,,,"is a removable plate forming the lid over the opening of a manhole, an opening large enough for a person to pass through that is used as an access point for an underground vault or pipe.",https://www.theworldavatar.com/kg/ontosewage/,Cover
+CoverClass,Class,,,,,,is the type of manhole's cover.,https://www.theworldavatar.com/kg/ontosewage/,Cover class
+FictionalShaft,Class,CoverClass,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Fictional shaft

```

Comment:
I do not think that makes sense. I believe `Fictional` was included here in the initial data model to re-emphasise that the shaft is fictional and, hence, has no cover. Ideally, we would just not add such a triple then instead of having `Fictional` as subclass

Reply:
`FictionalShaft` sub-class is changed to `Fictional`.

Reply:
Sorry for not making may point clear enough: I believe they used `Fictional` as cocer class to set the cover class for fictional shafts (as relational databases might require some sort of entry, even if they are not very meaningful). In a graph, on the contrary, we might just decide not to instantiate this triple at all. This would mean we do not need the `Fictional` class and for all fictional shafts we simply do not instantiate any cover class, which seems more sensible then instantiating a fictional cover class (at least to me). Is that clearer?